staging/src/k8s.io/apimachinery/pkg/test/api_meta_scheme_test.go

```diff
@@ -68,7 +68,7 @@ func validateListType(target reflect.Type) error {
 // TestListTypes verifies that no external type violates the api convention of
 // list types.
 func TestListTypes(t *testing.T) {
```

Comment:
This is neat. This is actually a reflective test of registered types.  Leave this one in location and we'll sort out what do do about it separately.